[2025-02-14T20:05:33.577+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T20:05:33.615+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt-etl-dag.create_missing_tables manual__2025-02-14T20:05:20.074683+00:00 [queued]>
[2025-02-14T20:05:33.630+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt-etl-dag.create_missing_tables manual__2025-02-14T20:05:20.074683+00:00 [queued]>
[2025-02-14T20:05:33.631+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-14T20:05:33.652+0000] {taskinstance.py:2890} INFO - Executing <Task(SnowflakeOperator): create_missing_tables> on 2025-02-14 20:05:20.074683+00:00
[2025-02-14T20:05:33.662+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=142) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T20:05:33.665+0000] {standard_task_runner.py:72} INFO - Started process 146 to run task
[2025-02-14T20:05:33.666+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dbt-etl-dag', 'create_missing_tables', 'manual__2025-02-14T20:05:20.074683+00:00', '--job-id', '185', '--raw', '--subdir', 'DAGS_FOLDER/dbt-etl.py', '--cfg-path', '/tmp/tmp4qxm53tu']
[2025-02-14T20:05:33.670+0000] {standard_task_runner.py:105} INFO - Job 185: Subtask create_missing_tables
[2025-02-14T20:05:33.727+0000] {task_command.py:467} INFO - Running <TaskInstance: dbt-etl-dag.create_missing_tables manual__2025-02-14T20:05:20.074683+00:00 [running]> on host aff10611ebf2
[2025-02-14T20:05:33.828+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt-etl-dag' AIRFLOW_CTX_TASK_ID='create_missing_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T20:05:20.074683+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T20:05:20.074683+00:00'
[2025-02-14T20:05:33.830+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T20:05:33.845+0000] {sql.py:295} INFO - Executing: 
CREATE TABLE IF NOT EXISTS Balance_Sheet (
    adsh STRING(20) NOT NULL,
    cik INT NOT NULL,
    company_name STRING(150),
    ticker STRING(10),
    sic STRING(4),
    filing_date DATE NOT NULL,
    fiscal_year INT NOT NULL,
    fiscal_period STRING(10) NOT NULL,
    tag STRING(256) NOT NULL,
    description STRING(512),
    ddate DATE NOT NULL,
    value DECIMAL(28,4),
    uom STRING(20),
    segment STRING(1024),
    source STRING(10),
    PRIMARY KEY (adsh, tag, ddate)
);

CREATE TABLE IF NOT EXISTS Income_Statement (
    adsh STRING(20) NOT NULL,
    cik INT NOT NULL,
    company_name STRING(150),
    ticker STRING(10),
    sic STRING(4),
    filing_date DATE NOT NULL,
    fiscal_year INT NOT NULL,
    fiscal_period STRING(10) NOT NULL,
    tag STRING(256) NOT NULL,
    description STRING(512),
    ddate DATE NOT NULL,
    value DECIMAL(28,4),
    uom STRING(20),
    segment STRING(1024),
    source STRING(10),
    PRIMARY KEY (adsh, tag, ddate)
);

CREATE TABLE IF NOT EXISTS Cash_Flow (
    adsh STRING(20) NOT NULL,
    cik INT NOT NULL,
    company_name STRING(150),
    ticker STRING(10),
    sic STRING(4),
    filing_date DATE NOT NULL,
    fiscal_year INT NOT NULL,
    fiscal_period STRING(10) NOT NULL,
    tag STRING(256) NOT NULL,
    description STRING(512),
    ddate DATE NOT NULL,
    value DECIMAL(28,4),
    uom STRING(20),
    segment STRING(1024),
    source STRING(10),
    PRIMARY KEY (adsh, tag, ddate)
);

[2025-02-14T20:05:33.859+0000] {base.py:84} INFO - Retrieving connection 'snowflake_default'
[2025-02-14T20:05:33.871+0000] {base.py:84} INFO - Retrieving connection 'snowflake_default'
[2025-02-14T20:05:33.873+0000] {connection.py:424} INFO - Snowflake Connector for Python Version: 3.13.2, Python Version: 3.12.9, Platform: Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-02-14T20:05:33.874+0000] {connection.py:1213} INFO - Connecting to GLOBAL Snowflake domain
[2025-02-14T20:05:33.875+0000] {connection.py:1295} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-02-14T20:05:34.741+0000] {sql.py:511} INFO - Running statement: CREATE TABLE IF NOT EXISTS Balance_Sheet (
    adsh STRING(20) NOT NULL,
    cik INT NOT NULL,
    company_name STRING(150),
    ticker STRING(10),
    sic STRING(4),
    filing_date DATE NOT NULL,
    fiscal_year INT NOT NULL,
    fiscal_period STRING(10) NOT NULL,
    tag STRING(256) NOT NULL,
    description STRING(512),
    ddate DATE NOT NULL,
    value DECIMAL(28,4),
    uom STRING(20),
    segment STRING(1024),
    source STRING(10),
    PRIMARY KEY (adsh, tag, ddate)
);, parameters: None
[2025-02-14T20:05:34.922+0000] {sql.py:520} INFO - Rows affected: 1
[2025-02-14T20:05:34.923+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/snowflake/hooks/snowflake.py:400: DeprecationWarning: Import of fetch_all_handler from the '***.providers.common.sql.hooks' module is deprecated and will
be removed in the future. Please import it from '***.providers.common.sql.hooks.handlers'.
  result = self._make_common_data_structure(handler(cur))  # type: ignore[attr-defined]

[2025-02-14T20:05:34.924+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/snowflake/hooks/snowflake.py:401: DeprecationWarning: Import of return_single_query_results from the '***.providers.common.sql.hooks' module is deprecated and will
be removed in the future. Please import it from '***.providers.common.sql.hooks.handlers'.
  if return_single_query_results(sql, return_last, split_statements):

[2025-02-14T20:05:34.924+0000] {snowflake.py:409} INFO - Rows affected: 1
[2025-02-14T20:05:34.925+0000] {snowflake.py:410} INFO - Snowflake query id: 01ba64d5-0004-9ece-0003-d5630012b2a6
[2025-02-14T20:05:34.925+0000] {sql.py:511} INFO - Running statement: CREATE TABLE IF NOT EXISTS Income_Statement (
    adsh STRING(20) NOT NULL,
    cik INT NOT NULL,
    company_name STRING(150),
    ticker STRING(10),
    sic STRING(4),
    filing_date DATE NOT NULL,
    fiscal_year INT NOT NULL,
    fiscal_period STRING(10) NOT NULL,
    tag STRING(256) NOT NULL,
    description STRING(512),
    ddate DATE NOT NULL,
    value DECIMAL(28,4),
    uom STRING(20),
    segment STRING(1024),
    source STRING(10),
    PRIMARY KEY (adsh, tag, ddate)
);, parameters: None
[2025-02-14T20:05:35.147+0000] {sql.py:520} INFO - Rows affected: 1
[2025-02-14T20:05:35.148+0000] {snowflake.py:409} INFO - Rows affected: 1
[2025-02-14T20:05:35.148+0000] {snowflake.py:410} INFO - Snowflake query id: 01ba64d5-0004-9ecd-0003-d5630013432a
[2025-02-14T20:05:35.149+0000] {sql.py:511} INFO - Running statement: CREATE TABLE IF NOT EXISTS Cash_Flow (
    adsh STRING(20) NOT NULL,
    cik INT NOT NULL,
    company_name STRING(150),
    ticker STRING(10),
    sic STRING(4),
    filing_date DATE NOT NULL,
    fiscal_year INT NOT NULL,
    fiscal_period STRING(10) NOT NULL,
    tag STRING(256) NOT NULL,
    description STRING(512),
    ddate DATE NOT NULL,
    value DECIMAL(28,4),
    uom STRING(20),
    segment STRING(1024),
    source STRING(10),
    PRIMARY KEY (adsh, tag, ddate)
);, parameters: None
[2025-02-14T20:05:35.325+0000] {sql.py:520} INFO - Rows affected: 1
[2025-02-14T20:05:35.326+0000] {snowflake.py:409} INFO - Rows affected: 1
[2025-02-14T20:05:35.326+0000] {snowflake.py:410} INFO - Snowflake query id: 01ba64d5-0004-9ecc-0003-d563001332d6
[2025-02-14T20:05:35.327+0000] {connection.py:803} INFO - closed
[2025-02-14T20:05:35.462+0000] {connection.py:809} INFO - No async queries seem to be running, deleting session
[2025-02-14T20:05:35.611+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/snowflake/hooks/snowflake.py:419: DeprecationWarning: Import of return_single_query_results from the '***.providers.common.sql.hooks' module is deprecated and will
be removed in the future. Please import it from '***.providers.common.sql.hooks.handlers'.
  if return_single_query_results(sql, return_last, split_statements):

[2025-02-14T20:05:35.612+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/operators/sql.py:311: DeprecationWarning: Import of return_single_query_results from the '***.providers.common.sql.hooks' module is deprecated and will
be removed in the future. Please import it from '***.providers.common.sql.hooks.handlers'.
  if return_single_query_results(self.sql, self.return_last, self.split_statements):

[2025-02-14T20:05:35.649+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T20:05:35.650+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dbt-etl-dag, task_id=create_missing_tables, run_id=manual__2025-02-14T20:05:20.074683+00:00, execution_date=20250214T200520, start_date=20250214T200533, end_date=20250214T200535
[2025-02-14T20:05:35.731+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T20:05:35.757+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-14T20:05:35.761+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
